src/app/heroes/heroes.component.ts

```diff
@@ -10,14 +11,13 @@ import { MessageService } from '../message.service';
 })
 export class HeroesComponent implements OnInit {
 
-  heroes: Hero[] = [];
+  public heroes$!: Observable<Hero[]>;
```

Comment:
Access modifier is not required. Its public by default.

You might want to consider just initializing it when the component is created, i.e. 
```ts
heroes$ = this.heroService.getHeroes();
```

Reply:
Done